status-im/status-react
# update for testing local pairing

### Summary

This PR is used for testing purpose that testing the feature local pairing since currently `status-go-version` status-mobile used on branch `develop` doesn't contains the corresponding feature [Sync all devices after initial pairing](https://github.com/status-im/status-mobile/issues/14447).

[Relate PR](https://github.com/status-im/status-mobile/pull/14514) for status-mobile. 
[Relate PR](https://github.com/status-im/status-go/pull/3047) for status-go.

To make sure local pairing works as expected, we should request a manual QA. 

### Testing notes
- it should worked on two different devices
- we should get all the data(profile picture/contact/display name/chat/chat message read/clear history/community/bookmark/setting/trusted user/verification request/wallet/saved address etc that `Sync all devices` support) on the receiver side in **LAN**

#### Platforms
- Android
- iOS

### Steps to test
- Open Status on Sender Device
- Create a new account and generate some data that `Sync all devices` support
- Scroll down to settings page and tap on Syncing menu item
- Tap on Sync new device
- Tap on Setup Syncing
- Enter the password of your account
- You will now see a QR code
- Open Status on Receiver Device
- Tap on 3 dots on top right on login screen
- Tap on scan sync code
- Scan the QR code which you see on Sender Device
- Wait for a few seconds for the Receiver Device, after paring successfully, it should be login automatically
- Check the data on Receiver Device

status: ready
